Jenkinsfile

```diff
@@ -61,6 +79,24 @@ pipeline {
             }
         }
 
+        stage('Publish ocm-2.3') {
```

Comment:
I would like not to duplicate stages... Let's use the same stage and adjust the variables and expressions

Reply:
Hard to do in declarative pipeline syntax without a pipeline library, but I'll throw it in a script block I guess.